spec/features/posts_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'rails_helper'
+
+RSpec.describe 'Post', type: :feature do
```

Comment:
- [ ] All your post tests are failing. Kindly fix them.

  You could have some users and posts on your local PC but there is no way for me to have them, as your migration files do not have any queries to create users and posts, only update. Also note that, by convention, seed data are created in the `db/seeds.rb` file, not migration files.

  ![image](https://github.com/agneta1991/social-media-app/assets/36542986/92dcd696-3d5e-4c5f-a9a9-bfc1d86252fb)

- [ ] **[OPTIONAL]** I think it would be better to split these show and index page tests into different files for better organization.